docs/components/DocumentationSidebarRightLink.tsx

```diff
@@ -140,13 +150,15 @@ const DocumentationSidebarRightLink = React.forwardRef<HTMLAnchorElement, Sideba
           style={{ paddingLeft }}
           href={'#' + slug}
           onClick={onClick}
-          css={[
-            STYLES_LINK,
-            isNested && STYLES_LINK_HEADER,
-            isCode && STYLES_LINK_CODE,
-            isActive && STYLES_LINK_ACTIVE,
-          ]}>
-          {displayTitle}
+          css={[STYLES_LINK, isNested && STYLES_LINK_HEADER, isActive && STYLES_LINK_ACTIVE]}>
+          <span css={[STYLES_LINK_LABEL, isCode && STYLES_LINK_CODE]}>{displayTitle}</span>
+          {tags && tags.length ? (
+            <div css={css({ display: 'inline-flex' })}>
+              {tags.map((tag: string) => (
```

Comment:
```suggestion
              {tags.map(tag => (
```
There's no need to type this manually (TypeScript knows it's a string)